v2/website/pages/index.js

```diff
@@ -112,16 +113,19 @@ const QUOTES = [
 
 function Home() {
   const [featureIndex, setFeatureIndex] = useState(0);
-  useEffect(() => {
-    const timer = window.setInterval(() => {
-      setFeatureIndex(
-        prevFeatureIndex => (prevFeatureIndex + 1) % FEATURES.length,
-      );
-    }, FEATURE_INTERVAL);
-    return () => {
-      window.clearInterval(timer);
-    };
-  }, []);
+  useEffect(
+    () => {
+      const timer = window.setTimeout(() => {
```

Comment:
Changing to `setTimeout` here and remount when `featureIndex` changes so that when user clicks on an item the timer is reset.